src/components/Reports/ReportSettingsEditForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Container, Grid, useTheme } from '@mui/material';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+import { useOrganization } from 'src/providers';
+import PageForm from 'src/components/common/PageForm';
+import { APP_PATHS } from 'src/constants';
+import ReportSettingsService, { ReportsSettings } from 'src/services/ReportSettingsService';
+import useSnackbar from 'src/utils/useSnackbar';
+import ReportSettingsEditFormFields from './ReportSettingsEditFormFields';
+
+interface ReportSettingsEditFormProps {
+  reportsSettings: ReportsSettings;
+  isEditing: boolean;
+}
+
+const ReportSettingsEditForm = ({ reportsSettings, isEditing }: ReportSettingsEditFormProps) => {
+  const theme = useTheme();
+  const { isMobile } = useDeviceInfo();
+  const { selectedOrganization } = useOrganization();
+  const history = useHistory();
+  const snackbar = useSnackbar();
+
+  const [localReportsSettings, setLocalReportsSettings] = useState(reportsSettings);
+
+  const onSave = useCallback(async () => {
+    const result = await ReportSettingsService.updateSettings({
```

Comment:
this is fine, wondering if using redux has any benefit here

Reply:
I thought about that, but it seemed like there would be a bunch of extra boilerplate for no added benefit.

Reply:
Got it, sounds fair to me.